tests/utils.py

```diff
@@ -175,10 +176,11 @@ def create_dummy_data_dict(self, path_to_dummy_data, data_url):
         dummy_data_dict = {}
         for key, abs_path in data_url.items():
             # we force the name of each key to be the last file / folder name of the url path
+            # if the url has arguments, we need to encode them with urllib.parse.quote_plus
             if isinstance(abs_path, list):
-                value = [os.path.join(path_to_dummy_data, x.split("/")[-1]) for x in abs_path]
+                value = [os.path.join(path_to_dummy_data, urllib.parse.quote_plus(x.split("/")[-1])) for x in abs_path]
```

Comment:
I guess ok for me...Over the course of adding all these datasets we have been adding more and more fixes to the MockDlManager to make it work in all cases to a point where it is now quite hard for a user to correctly name the `dummy_data` folder structure. 

One idea would be to write a `create_dumm_data_folder` command that does that automatically for the user. Or maybe we have to rethink the architecture / logic here a bit...

Reply:
What do you think @lhoestq ?

Reply:
This is probably what we will end up doing indeed